loader/loader.go

```diff
@@ -729,12 +730,6 @@ func (l *Loader) Restore(ctx context.Context) error {
 		return err2
 	}
 
-	l.wg.Add(1)
-	go func() {
-		defer l.wg.Done()
-		l.PrintStatus(ctx)
-	}()
-
```

Comment:
I think we can have a goroutine keep printing loader's status to users know loader work well now.

Reply:
Now the SourceWorker will periodically call `Unit.Status`, and `Unit.Status` will print unit's status to log

Reply:
I see

Reply:
> Now the SourceWorker will periodically call Unit.Status, and Unit.Status will print unit's status to log

this hide in `StatusJSON` ....  it's too hard to find, maybe change `StatusJSON` to `GetUnitAndSourceStatusJSON` or some other easy-for-read name  is better 😂

Reply:
https://github.com/pingcap/dm/pull/2076/commits/7dc8fc3b53064d9fe63b52e05aac60e8287dc670